ntoskrnl/fsrtl/fastio.c

```diff
@@ -1575,16 +1575,13 @@ FsRtlAcquireFileExclusiveCommon(IN PFILE_OBJECT FileObject,
     FilterCallbacks = DriverExtension->FsFilterCallbacks;
 
     /* Check if Filter Cllbacks are supported */
-    if (FilterCallbacks && FilterCallbacks->PreAcquireForSectionSynchronization)
+    if (FilterCallbacks && FilterCallbacks->SizeOfFsFilterCallbacks == sizeof(FS_FILTER_CALLBACKS) &&
```

Comment:
```suggestion
    if (FilterCallbacks &&
        FilterCallbacks->SizeOfFsFilterCallbacks >= RTL_SIZEOF_THROUGH_FIELD(FS_FILTER_CALLBACKS, PostAcquireForSectionSynchronization) &&
```

Reply:
Or, alternatively a `RTL_CONTAINS_FIELD(FilterCallbacks, FilterCallbacks->SizeOfFsFilterCallbacks, PostAcquireForSectionSynchronization)` macro

Reply:
Thanks. The 1st option looks shorter. It doesn't matter which from both to use, or has 2nd one an advantages, if any? I already committed the 1st, but it's not a problem to commit the 2nd if needed. :slightly_smiling_face: